chef/cookbooks/dhcp/recipes/default.rb

```diff
@@ -164,6 +164,6 @@
     end
   end
   supports :restart => true, :status => true, :reload => true
-  action :enable
+  action node[:provisioner][:enable_pxe] ? ["enable", "start"] : ["disable", "stop"]
```

Comment:
> I assume we could skip large parts of provisioner::update_nodes.rb and dhcp::default.rb did you already investigate that? It might be worth do that in the separate PR after we branched Tex, if it's too heavy for the Tex.

I disabled ```provisioner::dhcp_update.rb``` and ```provisioner::update_nodes.rb``` when PXE is disabled. 
```dhcp::default.rb``` is a bit more intrusive because this enables/disables the service at the end of the recipe. I skipped that for Tex. Can be done after branching but imho isn't really important.